builders/xcm/xcm-sdk/reference.md

```diff
@@ -0,0 +1,65 @@
+
+|                                   Method                                    |                                                                                     Description                                                                                      |
+|:---------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|    [`deposit()`](/builders/xcm/xcm-sdk/xcm-sdk/#deposit){target=_blank}     |                             Initiates a deposit to transfer assets from another chain to Moonbeam. **Must be called first before you can call `from()`**                             |
+|       [`from()`](/builders/xcm/xcm-sdk/xcm-sdk/#from){target=_blank}        |                                  Sets the source chain where the deposit will originate from. **Must be called first before you can call `get()`**                                   |
+|    [`get()`](/builders/xcm/xcm-sdk/xcm-sdk/#get-deposit){target=_blank}     | Sets the account on Moonbeam to deposit the funds to and the source account where the deposit will be sent from. **Must be called first before you can call `send()` or `getFee()`** |
+|    [`send()`](/builders/xcm/xcm-sdk/xcm-sdk/#send-deposit){target=_blank}    |                                                               Sends the deposit transfer data given an amount to send                                                                |
+| [`getFee()`](/builders/xcm/xcm-sdk/xcm-sdk/#get-fee-deposit){target=_blank} |                        Returns an estimate of the fee for transferring a given amount, which will be paid in the asset specified in the `deposit()` function                         |
+
+## Withdraw Methods {: #withdraw-methods }
+
+When building the transfer data needed for a withdraw, you'll use multiple methods to build the underlying XCM message and send it:
+
+|                                    Method                                    |                                                                  Description                                                                  |
+|:----------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------:|
+|    [`withdraw()`](/builders/xcm/xcm-sdk/xcm-sdk/#withdraw){target=_blank}    |          Initiates a withdraw to transfer assets from Moonbeam to another chain. **Must be called first before you can call `to()`**          |
+|          [`to()`](/builders/xcm/xcm-sdk/xcm-sdk/#to){target=_blank}          |            Sets the destination chain where the assets will be withdrawn to. **Must be called first before you can call `get()`**             |
```

Comment:
same comment as before, `to` and others are inherited or returned by calling `withdraw`? Maybe an asterisk and a note at the bottom of the table?